science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -52,9 +51,11 @@ Täglich spenden etwa 15,4 Mio. Nutzende der CWA ihre Daten. So wurden bisher ca
 
 Insgesamt gab es ca. 27,4 Mio. übermittelte Ergebnisse von Antigen-Schnelltests (RAT), bei denen ca. 7,9 Mio. mindestens eine Risikobegegnung vorausging. In der folgenden Abbildung ist der zeitliche Verlauf der Positivenanteile für jede Art der Risikobenachrichtigung angegeben. Über die Datenspende werden erst seit dem 24. Juni 2021 Antigen-Schnelltest-Ergebnisse registriert.
 
+<figure class="figure-img">
 <img src="./ppa_risk_infection_ct_timeline_rat-1.png" class="figure-img img-fluid" alt="Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT)."/>
 <figcaption aria-hidden="true">Abb. 1: Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT).</figcaption>
 <br>
+</figure>
```

Comment:
`<br>` needs to be removed as it is illegal HTML inside `<figure>`

Same for all other instances.